news/2020-04-10-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "osu! Spotlights: New Team and League System"
+date: 2020-04-13 08:00:00 +0000
+---
+
+With our latest [Spotlight season] released very recently, we are underway reworking the osu! Spotlights from grounds up, starting with a brand new team and adding a whole new league system.
```

Comment:
Two different options, either one would work.  `the osu! Spotlights`  sounds weird because `osu! Spotlights` is a name (proper noun) not a normal noun. Either `the` should be removed, or `project` (or a similar noun) should be added.